cluster/saltbase/salt/kubelet/default

```diff
@@ -138,9 +138,9 @@
 {% if pillar.get('network_provider', '').lower() == 'opencontrail' %}
   {% set network_plugin = "--network-plugin=opencontrail" %}
 {% elif pillar.get('network_provider', '').lower() == 'cni' %}
-  {% set network_plugin = "--network-plugin=cni --network-plugin-dir=/etc/cni/net.d/" %}
+  {% set network_plugin = "--network-plugin=cni --network-plugin-dir=/etc/cni/net.d/ --cni-bin-dir=/opt/cni/bin" %}
 {%elif pillar.get('network_policy_provider', '').lower() == 'calico' and grains['roles'][0] != 'kubernetes-master' -%}
-  {% set network_plugin = "--network-plugin=cni --network-plugin-dir=/etc/cni/net.d/" %}
+  {% set network_plugin = "--network-plugin=cni --network-plugin-dir=/etc/cni/net.d/ --cni-bin-dir=/opt/cni/bin" %}
```

Comment:
In order to make CNI installation consistent across all distros, could we standardize on `/home/kubernetes/cni/` or similar?

See here: https://github.com/kubernetes/kubernetes/pull/38169#issuecomment-279028474

Basically, users can't write a DaemonSet which installs a CNI plugin successfully for both GCI and Debian if we choose something in `/opt` for this flag.